src/java/com/zimbra/qa/selenium/projects/admin/ui/FormEditDistributionList.java

```diff
@@ -50,7 +50,12 @@
 		public static final String ALIAS_NAME = "css=input[id^='zdlgv__EDIT_ALIAS']";
 		public static final String ALIAS_DOMAIN_NAME = "css=input[id$='_name_3_display']";
 		public static final String zdlg_OK="css=td[id$='_button2_title']:contains('OK')";
-		
+		public static final String DirectMemberOf = "css=td[id$='_directMemberList___container'] div[id$='__rows']";
+		public static final String IndirectMemberOf = "css=td[id$='_indirectMemberList___container'] div[id$='__rows']";
+		public static final String FIND_DL_TEXTBOX = "css=input[id='ztabv__UNDEFINE_query_2']";
```

Comment:
if you see any numeric values in id selector that usually means it is auto generated, which can change time to time
can we use some class selector which will not change?